src/onegov/api/path.py

```diff
@@ -28,7 +29,8 @@ def get_api_endpoint(
     endpoint: str,
     page: int = 0,
     extra_parameters: dict[str, Any] | None = None,
-) -> ApiEndpoint | AuthEndpoint:
+) -> ApiEndpoint[_M] | AuthEndpoint | Any:
+
```

Comment:
Without `Any`, this fails with

`error: Incompatible return value type (got "ApiEndpoint[_M]", expected "ApiEndpoint[_M] | AuthEndpoint")  [return-value]`

I am not quite sure what's the issue

Reply:
This should be `ApiEndpoint[Any]`, unbound type variables are generally to be avoided, there's some rare edge cases where they can be useful.